R-packages/evalcast/R/get_predictions.R

```diff
@@ -88,6 +117,10 @@ get_predictions_single_date <- function(forecaster,
                       geo_type = geo_type,
                       geo_values = geo_values)
     })
+  if(!is.null(apply_corrections)){
+    corrected <- apply_corrections(df)
+    df$value = corrected$corrected
```

Comment:
If a user-supplied `apply_corrections` were to reorder the rows, then this could be a problem.  Perhaps a `left_join` would be safer?

Reply:
I made a number of changes to address this, adding checks and a test. Currently all tests fail due to the `covidcast::name_to_abbr()` bug (see #144, fixed by @ryantibs on #146 ).